src/coord/src/coord.rs

```diff
@@ -895,13 +895,10 @@ where
                 // a Kafka topic) that's been created on our behalf. If
                 // we fail now, we'll leak that external state.
```

Comment:
Details in top-level comment, but can you tone down this comment a bit? Maybe "failing from here on out is very unfortunate, because..."